code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -54,14 +54,23 @@
 	list_reagents = list("omnizine" = 30)
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ACID_PROOF
 
+/obj/item/reagent_containers/hypospray/emergency_combat
+	name = "emergency combat stabilizer" //Used for the new combat medical kit.
+	desc = "A small modified air-needle auto injector, used by support operatives to quickly stabilize their allies in critical condition."
+	amount_per_transfer_from_this = 5
+	icon_state = "hypo"
+	volume = 35
+	ignore_flags = 1 // So they can heal their comrades.
+	list_reagents = list("inacusiate" = 5, "stimulants" = 15, "atropine" = 15)
+
 /obj/item/reagent_containers/hypospray/combat
 	name = "combat stimulant injector"
 	desc = "A modified air-needle autoinjector, used by support operatives to quickly heal injuries in combat."
 	amount_per_transfer_from_this = 10
 	icon_state = "combat_hypo"
 	volume = 90

```

Comment:
The volume doesn't match the contained reagents total volume anymore.

Reply:
That' on purpose. You can fill any injectors with chemicals, by just clicked on them. I thought that's self-explanatory.